1_3.js

```diff
@@ -0,0 +1,5 @@
+function solution(s){
+    var answer = s.toLowerCase().split("p").length === s.toLowerCase().split("y").length
+        
+    return answer 
```

Comment:
```suggestion
     return s.toLowerCase().split("p").length === s.toLowerCase().split("y").length;
```
변수에 안담고 리턴 가능합니다.
문장 끝나면 세미콜론(;) 붙여서 컨벤션 지켜주세요

Reply:
컨벤션 신경쓰겠습니다